proxy/proxy.go

```diff
@@ -533,6 +539,26 @@ func callWeaveAttach(container *docker.Container, args []string) error {
 	return nil
 }
 
+func validateCIDRs(cidrs []string) error {
+	check := func(cidr string, prefixes []string) bool {
+		for _, prefix := range prefixes {
+			if strings.HasPrefix(cidr, prefix) {
+				if _, err := address.ParseCIDR(strings.TrimPrefix(cidr, prefix)); err == nil {
+					return true
+				}
+			}
+		}
+		return false
+	}
+
+	for _, cidr := range cidrs {
+		if cidr != "net:default" && !check(cidr, []string{"ip:", "net:", ""}) {
```

Comment:
According to https://github.com/weaveworks/weave/blob/bf8c8833e5f07ce5911f40e4e1d13f936dd9b508/weave#L275 , "net:default" is a valid argument.